validity/src/proposer.rs

```diff
@@ -188,74 +199,136 @@ where
                 return Ok(());
             }
         };
-
-        // Get all active (non-failed) requests with the same commitment config and start block >=
-        // latest_proposed_block_number. These requests are non-overlapping.
-        let mut requests = self
-            .driver_config
-            .driver_db_client
-            .fetch_ranges_after_block(
-                &[
-                    RequestStatus::Unrequested,
-                    RequestStatus::WitnessGeneration,
-                    RequestStatus::Execution,
-                    RequestStatus::Prove,
-                    RequestStatus::Complete,
-                ],
+    
+        let gas_threshold_u64 = self.program_config.gas_threshold;
```

Comment:
Why include the type in the var name?

Reply:
Because it is later converted to gat_threshold being a i64. This gas_threshold_u64 is just a temporal variable. I may use u64 everywhere to make it simpler.